src/lib.rs

```diff
@@ -552,6 +648,10 @@ pub mod pallet {
 		TooManyContributors,
 		/// Provided vesting period is not valid
 		VestingPeriodNonValid,
+		/// User provided a signature from a non-contributor relay account
+		NonContributedAddressProvided,
+		/// User submitted an unsifficient number of proofs to change the reward address
+		UnsifficientNumberOfValidProofs,
```

Comment:
```suggestion
		InsufficientNumberOfValidProofs,
```

Reply:
I thought I had changed this.Maybe it got messed with the merge with master

Reply:
I thought you had changed it too. It is a few places in the code.